### DISCUSSION TITLE
Arm/Raspberry Pi version

### DISCUSSION THREAD

**Discussion by wacra** [twesterapp/twester#44] [Ideas]
I would love to be able to just setup a pi to run it 24/7 without me having to leave my computer/laptop on all the time.

**Comment by ceoshikhar** [comment]
I don't have much idea about Raspberry Pi's and how they work. If you can run a GUI Linux on it then you can try running it. But I would recommend you to checkout [Twitch-Channel-Points-Miner-v2](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2). It's a CLI application written in Python that can probably help you with your problem. Twester is heavily dependent and inspired from it.
